src/content/structured/components/cards/code.mdx

```diff
@@ -309,16 +447,54 @@ export const topImage = [
   subheading="Card subheading"
   message="This is an example of a card with a top image."
   style={{width: "360px"}}>
-  <SlottedSVG slot="image-top" ... >
-    ...
-  </SlottedSVG>
-  <SlottedSVG slot="icon" ... >
-    ...
-  </SlottedSVG>
-  <IcButton variant="icon" title="More information" slot="interaction-button">
-    <SlottedSVG slot="icon" ... >
-      ...
+    <SlottedSVG
+      slot="image-top"
+      style={{
+        height: "100%",
+        width: "100%",
+        maxHeight: "20.375rem",
+        maxWidth: "20.375rem",
+      }}
+      viewBox="0 0 1600 1250"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <rect fill="#ff7700" width="1600" height="1600" y="-350" />
+      <polygon fill="#cc0000" points="957 450 539 900 1396 900" />
+      <polygon fill="#aa0000" points="957 450 872.9 900 1396 900" />
+      <polygon fill="#c50022" points="-60 900 398 662 816 900" />
+      <polygon fill="#a3001b" points="337 900 398 662 816 900" />
+      <polygon fill="#be0044" points="1203 546 1552 900 876 900" />
+      <polygon fill="#9c0036" points="1203 546 1552 900 1162 900" />
+      <polygon fill="#b80066" points="641 695 886 900 367 900" />
+      <polygon fill="#960052" points="587 900 641 695 886 900" />
+      <polygon fill="#b10088" points="1710 900 1401 632 1096 900" />
+      <polygon fill="#8f006d" points="1710 900 1401 632 1365 900" />
+      <polygon fill="#aa00aa" points="1210 900 971 687 725 900" />
+      <polygon fill="#880088" points="943 900 1210 900 971 687" />
+    </SlottedSVG>
+    <SlottedSVG>
+      slot="icon"
+      xmlns="http://www.w3.org/2000/svg"
+      height="24px"
+      viewBox="0 0 24 24"
+      width="24px"
+      fill="#000000"
+    >
+      <path d="M0 0h24v24H0V0z" fill="none" />
+      <path d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z" />
     </SlottedSVG>
+    <icButton variant="icon" title="More information" slot="interaction-button">
+      <SlottedSVG>
```

Comment:
```suggestion
      <SlottedSVG
```

Reply:
rolled back changes outside scope of ticket- this no longer applies.